### PR TITLE
Shortcuts config fix

### PR DESCRIPTION
**Pull request by kevinthegreat1**
Also add `/g` as default shortcut for `/garden`.

### CODE REVIEW

**Comment by qtlunya** [comment]
> Also add `/g` as default shortcut for `/garden`.

That sounds like a bad idea, isn't it an alias of `/guild` by default?

**Comment by x-yingcan-x** [comment]
I prefer /pv for /party leave, instead of /profileviewer, but its hardcoded.
if I need it I'll create on my own, hardcode shortcuts is bad idea.

**Comment by Julienraptor01** [comment]
> > Also add `/g` as default shortcut for `/garden`.
> 
> That sounds like a bad idea, isn't it an alias of `/guild` by default?

It indeed is

**Comment by Julienraptor01** [comment]
> I prefer /pv for /party leave, instead of /profileviewer, but its hardcoded.
> if I need it I'll create on my own, hardcode shortcuts is bad idea.

Yeah i really think we should not provide any short form shortcut by default and let the users make those

File: src/main/java/de/hysky/skyblocker/skyblock/shortcut/Shortcuts.java [hunk 1]
```diff
@@ -82,6 +83,7 @@ private static void registerDefaultShortcuts() {
         commands.put("/s", "/skyblock");
         commands.put("/i", "/is");
         commands.put("/h", "/hub");
+        commands.put("/g", "/garden");
```
Comment (kevinthegreat1) [inline src/main/java/de/hysky/skyblocker/skyblock/shortcut/Shortcuts.java:86]:
```suggestion
        commands.put("/g", "/warp garden");
```

**Comment by kevinthegreat1** [comment]
> > Also add `/g` as default shortcut for `/garden`.
> 
> That sounds like a bad idea, isn't it an alias of `/guild` by default?

It does not affect `/g command`. Also Sorry I accidentally edited your comment.

**Comment by kevinthegreat1** [comment]
> I prefer /pv for /party leave, instead of /profileviewer, but its hardcoded. if I need it I'll create on my own, hardcode shortcuts is bad idea.

This does not added hardcoded shortcuts and only applies to new users.

**Review by AzureAaron — APPROVED** [review]
seems ok

**Pull request merged**